src/test_pyddx.py

```diff
@@ -58,9 +58,10 @@ def test_reference_pcm():
     state.solve()
     state.solve_adjoint()
 
-    force = state.solvation_force_terms()
+    force = state.solvation_force_terms(solute_field["e"])
```

Comment:
With my suggestion this would just be `state.solvation_force_terms(solute_field)` which would automatically work also for LPB because then the required derivative order is one higher, such that the default entries in the solute_field dict automatically contain a `"g"` entry. So less worry for the user.